WhaleSpotting/Controllers/UserController.cs

```diff
@@ -28,4 +29,16 @@ public IActionResult GetById([FromRoute] int userId)
             return NotFound();
         }
     }
-}
\ No newline at end of file
+    [HttpPost("create")]
+    public IActionResult Create([FromBody] UserRequest newUser)
+    {
+        if (!ModelState.IsValid)
+        {
+            return BadRequest(ModelState);
+        }
+        var newuser = _userService.Create(newUser);
+        var url = Url.Action("GetById", new { userId = newuser.Id });
+        var responseViewModel = new UserResponse(newuser);
+        return Created(url, responseViewModel);
```

Comment:
Sorry - it might not have been clear before. We can change this to something like this
```suggestion
        var user = _userService.Create(newUser);
        var url = Url.Action("GetById", new { userId = newuser.Id });
        return Created(url, new UserResponse(user));
```

Reply:
changes made